### ISSUE TITLE
Mod being removed from mods folder upon world load

### ISSUE THREAD

**Issue by 5aturat3dmark** [JBurlison/Pandaros.Settlers#50]
Affected Version: multiple / not specific

Upon loading into the world, it can be observed in the \gamedata\mods folder that the Settlers folder is removed completely from the directory, thus loading the world without the mod installed and causing missing block and texture issues.

I've tested this with previous version of the mod (all the way back to 0.8.1.2) and had the same result of the mode being deleted as soon as the load button is selected in the main menu.

What confused me the most was that I was on 0.8.1.2 yesterday on 4 May, saw the updates and updated to 0.8.1.4 where the issue started, as I saw more missing textures and error textures for items in the stock pile. But it continues, regardless of which version i load, including 0.8.1.2 which worked perfectly fine before.

One interesting thing i noticed was that when i rename the 'Pandaros' folder in the 'mods' folder to anything else (Pandaros.0.8.1.4 and such), the mod was not removed at all and when loading the world out, i only encountered some missing block and item textures.

I thought this might have been potential world corruption, but I created new worlds for each version just to make sure, and saw the same deletion being made each time. Especially after i renamed the file as previously mentioned, my world seemed mostly normal aside of the few texture issues.

**Comment by JBurlison** [comment]
Thanks for the report! I will take a look!

**Comment by 5aturat3dmark** [comment]
Hey wanted to update you that I loaded the new update into the mods folder and noticed that the issue is no longer occuring and that everything seems to be working normally (aside of the aforementioned missing colored clay textures). I'm not sure if it was something to do with the code changes that fixed it or just my game wigging out but it dosen't delete the folder anymore and I was able to just drop it in and load my world normally. Thanks for looking into it though!
